test/typescript/client.ts

```diff
@@ -26,5 +26,5 @@ const req = Client.createRequest({
 Client.request({
   url: "/test"
 }).then(res => {
-  res[0].headers
-})
\ No newline at end of file
+  res[0].statusCode;
```

Comment:
From `headers` to `statusCode`?

Reply:
Shouldn't matter, I was making sure the typescript response fields were populating properly. and that I had the right variable names.